client/src/components/Offer/OfferDetails.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styles from "./Offer.module.scss";
+import { getNiceSalary } from "./OfferHelper";
+
+type Props = {
+  position: string;
+  company: string;
+  tags?: string[];
+  date?: string | Date;
+  salary: {
+    from: number;
+    to: number;
+    currency: string;
+  };
+  location?: string;
+  isNew?: boolean;
+};
+
+const defaultCompanyLogo =
+  "https://www.gsr-technology.co.uk/wp-content/uploads/2015/10/partner-logo-placeholder-300x150.jpg";
+
+export const OfferDetails = ({
+  position,
+  company,
+  salary,
+  location,
+  isNew,
+  tags,
+  date,
+}: Props) => {
+  return (
+    <div className={styles.detailsContainer}>
+      <div className={styles.positionAndCompoany}>
+        <div className={styles.position}>{position}</div>
+        <div className={styles.company}>{company}</div>
+      </div>
+
+      <div className={styles.positionDescription}>
+        {tags && (
+          <div className={styles.tags}>
+            {tags.map((tag, index) => {
+              return (
+                <div key={index} className={styles.tag}>
+                  {tag}
+                </div>
+              );
+            })}
+          </div>
+        )}
+
+        <div className={styles.positionDetails}>
+          <div className={styles.salary}>{getNiceSalary(salary)}</div>
+          <div className={styles.date}>
+            <svg width="24px" height="24px" viewBox="0 0 24 24">
```

Comment:
Maybe use the material UI component: https://material-ui.com/components/icons/#icons